website/versioned_docs/version-5.x/ecosystem-alpinejs.md

```diff
@@ -0,0 +1,236 @@
+---
+id: ecosystem-alpinejs
```

Comment:
Please add this to the sidebars.json file (https://github.com/single-spa/single-spa.js.org/blob/master/website/versioned_sidebars/version-5.x-sidebars.json#L201), so that it appears in the sidebar

Reply:
done